### PR TITLE
Fix bug profile page

### PR DESCRIPTION
**Pull request by khanhnguyen1025**

### CODE REVIEW

File: src/app/pages/login/login.component.ts [hunk 1]
```diff
@@ -55,33 +52,34 @@ export class LoginComponent implements OnInit {
...
-    gapi.load('auth2', () => {
+    gapi.load("auth2", () => {
       this.auth2 = gapi.auth2.init({
-        clientId: '10558520426-5epndmc1a1dgsjvffftbvn60rr6521hh.apps.googleusercontent.com',
-        cookiepolicy: 'single_host_origin',
-        scope: 'profile email'
+        clientId:
+          "10558520426-5epndmc1a1dgsjvffftbvn60rr6521hh.apps.googleusercontent.com",
+        cookiepolicy: "single_host_origin",
+        scope: "profile email"
```
Comment (thanglenova) [inline src/app/pages/login/login.component.ts]: please use single quote `'`

File: src/app/pages/login/login.component.ts [hunk 2]
```diff
@@ -55,33 +52,34 @@ export class LoginComponent implements OnInit {
...
       });
-      this.attachSigin(document.getElementById('google-button'));
+      this.attachSigin(document.getElementById("google-button"));
```
Comment (thanglenova) [inline src/app/pages/login/login.component.ts]: please use single quote `'`

File: src/app/pages/login/login.component.ts [hunk 3]
```diff
@@ -55,33 +52,34 @@ export class LoginComponent implements OnInit {
...
-        console.log('Name: ' + profile.getName());
-        console.log('Email: ' + profile.getEmail());
-        this.authService.loginGoogle(googleUser.getAuthResponse().id_token)
-        .subscribe((data)=>{
-          this.ngZone.run(() =>  this.router.navigate(['/welcome'])).then();
-          let token = data.accessToken;
-          localStorage.setItem('currentUser',token);
+    this.auth2.attachClickHandler(element, {}, googleUser => {
+      let profile = googleUser.getBasicProfile();
+      console.log("Token|| " + googleUser.getAuthResponse().id_token);
```
Comment (thanglenova) [inline src/app/pages/login/login.component.ts]: please remove console log

File: src/app/pages/login/login.component.ts [hunk 4]
```diff
@@ -1,50 +1,47 @@
...
 
 declare const gapi: any;
 
 @Component({
-  selector: 'app-login',
-  templateUrl: './login.component.html',
-  styleUrls: ['./login.component.scss']
+  selector: "app-login",
+  templateUrl: "./login.component.html",
+  styleUrls: ["./login.component.scss"]
```
[the remark this answers is not in the document]
Comment (khanhnguyen1025) [reply src/app/pages/login/login.component.ts]: done

File: src/app/pages/login/login.component.ts [hunk 5]
```diff
@@ -1,50 +1,47 @@
...
 })
 export class LoginComponent implements OnInit {
-
   submitted = false;
   loading: boolean = false;
   returnUrl: string;
 
-  clientId: string = '10558520426-5epndmc1a1dgsjvffftbvn60rr6521hh.apps.googleusercontent.com';
+  clientId: string =
+    "10558520426-5epndmc1a1dgsjvffftbvn60rr6521hh.apps.googleusercontent.com";
```
[the remark this answers is not in the document]
Comment (khanhnguyen1025) [reply src/app/pages/login/login.component.ts]: done

File: src/app/pages/login/login.component.ts [hunk 6]
```diff
@@ -1,50 +1,47 @@
...
-    'https://www.googleapis.com/auth/plus.me',
-    'https://www.googleapis.com/auth/contacts.readonly',
-    'https://www.googleapis.com/auth/admin.directory.user.readonly'
-  ].join(' ');
+    "profile",
+    "email",
+    "https://www.googleapis.com/auth/plus.me",
+    "https://www.googleapis.com/auth/contacts.readonly",
+    "https://www.googleapis.com/auth/admin.directory.user.readonly"
+  ].join(" ");
```
[the remark this answers is not in the document]
Comment (khanhnguyen1025) [reply src/app/pages/login/login.component.ts]: done

File: src/app/pages/login/login.component.ts [hunk 7]
```diff
@@ -1,50 +1,47 @@
...
+    private alertService: AlertService,
+    public message: NzMessageService
   ) {
-
-    console.log('ElementRef: ', this.element);
-
     //redirect to home if logged
     if (this.authService.currentUser) {
-      this.router.navigate[('/welcome')];
+      this.router.navigate["/welcome"];
```
[the remark this answers is not in the document]
Comment (khanhnguyen1025) [reply src/app/pages/login/login.component.ts]: done

File: src/app/pages/login/login.component.ts [hunk 8]
```diff
@@ -55,33 +52,34 @@ export class LoginComponent implements OnInit {
   public auth2: any;
 
   public googleInit() {
-    gapi.load('auth2', () => {
+    gapi.load("auth2", () => {
```
[the remark this answers is not in the document]
Comment (khanhnguyen1025) [reply src/app/pages/login/login.component.ts]: done

File: src/app/pages/login/login.component.ts (hunk 1 above)
Comment (khanhnguyen1025) [reply src/app/pages/login/login.component.ts]: done

File: src/app/pages/login/login.component.ts (hunk 2 above)
Comment (khanhnguyen1025) [reply src/app/pages/login/login.component.ts]: done

File: src/app/pages/login/login.component.ts (hunk 3 above)
Comment (khanhnguyen1025) [reply src/app/pages/login/login.component.ts]: done

**Pull request merged**